ecosystem/sep-0006.md

```diff
@@ -540,6 +761,60 @@ The response should be a JSON object like:
       "fee_percent": 0
     }
   },
+  "deposit_indicative_quote": {
+    "USD": {
+      "enabled": true,
+      "authentication_required": true,
+      "fee_fixed": 5,
+      "fee_percent": 1,
+      "min_amount": 0.1,
+      "max_amount": 1000,
```

Comment:
I think we should remove these attributes for `deposit_indicative_quote` objects since the limits should be imposed on the asset sent by the user, which is the off-chain asset. Instead we can make `amount` required on the `deposit_indicative_quote` endpoint and enforce limits there and communicate those limits in 400 error messages.

Its not a great design, but `min_amount` and `max_amount` in terms of the on-chain asset can't be used until the anchor determines the exchange rate, which may not happen until after the user has sent the off-chain funds.

Reply:
That makes sense! Unless the value is dynamically defined from the Anchor side, but we don't want to impose that kind of a burden, right?

Removed those fields in https://github.com/stellar/stellar-protocol/pull/973/commits/ea0a84cd45149ffc6a757c6f96d25354e987802a.